package exception.compiletime;

import java.io.File;
import java.io.FileWriter;
import java.io.IOException;

public class Example2 {

	public static void main(String[] args) {
		
		File file = new File("D:/files12"); // Creating Object for file class
		FileWriter fileWriter = null;
		String s = null;
		try {
			fileWriter = new FileWriter(file+"/empnames.txt"); // opening connection for FileWriter
			int k = 10/0;
			int len = s.length();
			fileWriter.write("Ram\nRajesh\nRakesh\nAlex\nRaj"); // We are writing data into the file
			fileWriter.flush(); // flush is equal to in database commit (This is kind of giving permission)
			fileWriter.close(); // we are closing connection for FileWriter
			
		} catch (IOException e) {
			try {
				fileWriter.close();
			} catch (IOException e1) {
				e1.printStackTrace();
			}
			e.printStackTrace();
		}
		catch (ArithmeticException e) {
			try {
				fileWriter.close();
			} catch (IOException e1) {
				e1.printStackTrace();
			}
		}
		
		System.out.println("main method end...");
	}
}
